backend/src/main/java/net/finman/model/UserCustomer.java

```diff
@@ -0,0 +1,35 @@
+package net.finman.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class UserCustomer {
+
+    private String email;
+    private Customer customer;
+
+    public void setUser(String email) {
+        this.email = email;
+    }
+
+    public String getUser() {
+        return this.email;
+    }
```

Comment:
The setter might be the  reason why we need to use the key "user" instead of "email". Spring Boot might be matching the setters with the key's in the JSON instead of the actual attributes in the POJO. Try change the function name to `setEmail(...)` and `getEmail(...)`and see if that resolves the issue?